libshaderc/src/shaderc.cc

```diff
@@ -168,6 +168,7 @@ class StageDeducer {
   bool error_;
 };
 
+// A bridge between the libshaderc includer and libshaderc_util includer.
 class InternalFileIncluder : public shaderc_util::CountingIncluder {
```

Comment:
I think we should change the name 'InternalFileIncluder' to something else. Anyway, we can do that later in another CL.